### DISCUSSION TITLE
How do I adjust the cursor position?

### DISCUSSION THREAD

**Discussion by ru88s** [react-hook-form/react-hook-form#8150] [Q&A] [answered]
How do I adjust the cursor position?
I want to adjust the cursor in a tag like GitHub.
https://i.gyazo.com/16db19d262d06d606c3a4f78cb8e2e33.gif

**Comment by bluebill1049** [comment, marked as answer]
This is something out of the scope of this library. This article maybe helpful to you: https://bobbyhadz.com/blog/javascript-move-cursor-end-of-text-input

**Reply by ru88s** [reply]
@bluebill1049 
The setSelectionRange helped!
Thank you very much.

[HTMLInputElement\.setSelectionRange\(\) \- Web API \| MDN](https://developer.mozilla.org/ja/docs/Web/API/HTMLInputElement/setSelectionRange)
